xml/System.Data.OracleClient/OracleMonthSpan.xml

```diff
@@ -581,9 +581,9 @@
         <Parameter Name="x" Type="System.Data.OracleClient.OracleMonthSpan" />
       </Parameters>
       <Docs>
-        <param name="x">An <see cref="T:System.Data.OracleClient.OracleMonthSpan" /> structure to convert to an <see langword="Int32" />.</param>
-        <summary>Converts an <see cref="T:System.Data.OracleClient.OracleMonthSpan" /> structure to an <see langword="Int32" />.</summary>
-        <returns>An <see langword="Int32" />.</returns>
+        <param name="x">An <see cref="T:System.Data.OracleClient.OracleMonthSpan" /> structure to convert to an <see cref="T:System.Int32" />.</param>
```

Comment:
We usually just more descriptive text in these since the type is automatically shown. Maybe this?

```suggestion
        <param name="x">An <see cref="T:System.Data.OracleClient.OracleMonthSpan" /> structure to convert to an integer.</param>
```